helper/resource.js

```diff
@@ -0,0 +1,113 @@
+const { ResourceType } = require("../common/enum.utils");
```

Comment:
@ankitpws - Please add basic info at the beginning of the file similar to other services.